pom.xml

```diff
@@ -24,75 +24,161 @@
   THE SOFTWARE.
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-  <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>2.33</version>
```

Comment:
Supersedes #5 I suppose.

Reply:
Thanks for pointer.  I'm closing both this pull request and the preceding pull request.  I think you're correct that the best place to make these types of changes is in the maven archetype.

I've also updated the Jenkinsfile to show it testing two different versions (LTS from the pom, latest weekly in the Jenkinsfile).  I added an explanatory comment as well so that it was clear why the technique was being used.

See the [jenkins-world-2017 branch on my fork](This pull request is not useful.  As Jesse suggests in #2, we should update the maven archetype to include these types of changes, not make them in this repository.  The maven archetype provides the settings for initial class and more which are not available from a forked repository.

Refer to the [jenkins-world-2017 branch on my fork](https://github.com/MarkEWaite/hello-world-plugin/tree/jenkins-world-2017) for the latest changes I've made.) for those changes.